git/.yamllint

```diff
@@ -0,0 +1,23 @@
+extends: default
```

Comment:
Does this file allow for comment lines? I love that we can configure everything this way, but descriptions of what, and more importantly **why** a non-intuitive parameter is set such as it is are quite useful.

For example, why is `truthy` set to `disable`? Another quick question about that, is there a difference between `0` and `1` and `disable` and `enable`?

Follow up to that, which `yamllint` program are you using? On my laptop I actually have a script in `~/bin/` called `yamllint` that is just a crappy ruby interpreter that just tries to load the file given in `$1`.